src/test/unit/function-info.test.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { stripFunctionArguments } from '../../profile-logic/function-info';
+
+describe('get-function-name', function () {
+  it('should strip the function arguments', function () {
+    expect(stripFunctionArguments('ns::fn()')).toEqual('ns::fn');
+    expect(stripFunctionArguments('ns::fn(bool (*)(JS::Handle<JSObject*>))')).toEqual('ns::fn');
+    expect(stripFunctionArguments('Bar* ns::fn<(Foo)0>(void const*, unsigned int)')).toEqual('Bar* ns::fn<(Foo)0>');
```

Comment:
Here is the profile I use:

http://localhost:4242/public/f1c034199abaa5756b405cd3fbcc9d65c3f34c28/calltree/?callTreeFilters=prefix-04kBvx8CcCdx8LgCgx8ChCiC.hUx5PRiLZtX_1PRiLZtx8LS2PxtOGsz5M9USuUwgMaUU0UwiJx2xwU8ES3QJoUKpHx1CJfWElBy8&hiddenThreads=&react_perf&thread=0&threadOrder=0-2-3-4-5-1